sys/net/gnrc/transport_layer/tcp/Kconfig

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_TCP
+    bool "Configure GNRC_TCP"
+    depends on MODULE_GNRC_TCP
+    help
+        Configure the GNRC_TCP using Kconfig.
+
+if KCONFIG_MODULE_GNRC_TCP
+
+config GNRC_TCP_CONNECTION_TIMEOUT_DURATION
+    int "Timeout duration for user calls in microseconds"
+    default 120000000
+    help
+        Timeout duration for user calls. Default value is 120000000 microseconds
+        (2 minutes).
+
+config GNRC_TCP_MSL
+    int "Maximum segment lifetime (MSL) in microseconds"
+    default 30000000
+    help
+        Maximum segment lifetime (MSL) in microseconds. Default value is 30
+        seconds.
+
+config GNRC_TCP_MSS_MULTIPLICATOR
+    int "Number of MSS sized packets stored in receive buffer"
+    default 1
+    help
+        Configure MSS Multiplicator i.e. number of MSS sized packets stored in
+        receive buffer.
+
+config GNRC_TCP_RCV_BUFFERS
+    int "Number of preallocated receive buffers"
+    default 1
+
+config GNRC_TCP_RTO_LOWER_BOUND
+    int "Lower bound for RTO in microseconds"
+    default 1000000
+    help
+        Lower bound value for retransmission timeout (RTO) in microseconds.
+        Default value is 1000000 microseconds (1 second). Retransmission
+        timeout determines how long TCP waits for acknowledgment (ACK) of
+        transmitted segment. Refer to RFC 6298 for more information.
+
+config GNRC_TCP_RTO_UPPER_BOUND
+    int "Upper bound for RTO in microseconds"
+    default 60000000
+    help
+        Upper bound value for retransmission timeout (RTO) in microseconds.
+        Default value is 60000000 microseconds (60 seconds). Refer to RFC 6298
+        for more information.
+
+config GNRC_TCP_RTO_GRANULARITY
+    int "Clock granularity for RTO in microseconds"
+    default 10000
+    help
+        Clock granularity for retransmission timeout (RTO) for TCP in
+        microseconds. Default value is 10000000 microseconds (10 seconds). 
```

Comment:
This is not true. The default granularity is 10 miliseconds.

Reply:
Updated. Thanks